src/core_plugins/metrics/public/components/aggs/derivative.js

```diff
@@ -1,62 +1,58 @@
-import React, { Component, PropTypes } from 'react';
+import React, { PropTypes } from 'react';
 import AggSelect from './agg_select';
 import MetricSelect from './metric_select';
 import AggRow from './agg_row';
 import createChangeHandler from '../lib/create_change_handler';
 import createSelectHandler from '../lib/create_select_handler';
 import createTextHandler from '../lib/create_text_handler';
 
-class DerivativeAgg extends Component {
```

Comment:
If the class -> function conversion isn't necessary, I'd be fine with it still but another separate commit would be very helpful.